internal/backend/runtime/omni/infraprovider/state_test.go

```diff
@@ -473,6 +526,15 @@ func prepareInfraProviderServiceAccount(ctx context.Context) context.Context {
 	return ctx
 }
 
+func prepareRole(ctx context.Context, role role.Role) context.Context {
```

Comment:
I think it makes sense to unify with `prepareInfraProviderServiceAccount` so we don't create duplicate functions.

Reply:
👍 done